airframe-jmx/src/test/scala/wvlet/airframe/jmx/JMXRegistryTest.scala

```diff
@@ -70,6 +78,25 @@ class JMXRegistryTest extends AirSpec {
     }
   }
 
+  test("specify property name by annotation") {
+    val b = new NamedMBean
+    agent.register(b)
+
+    val mbeanServer = ManagementFactory.getPlatformMBeanServer
+    val mbean       = mbeanServer.getMBeanInfo(new ObjectName("wvlet.airframe.jmx:name=NamedMBean"))
+    val attrs       = mbean.getAttributes
+    attrs.length shouldBe 1
+    attrs(0).getName shouldBe "memory.free"
+
+    if (!JMXUtil.isAtLeastJava9) {
+      val m = agent.getMBeanInfo("wvlet.airframe.jmx:name=NamedMBean")
+      debug(m)
+
+      val a = agent.getMBeanAttribute("wvlet.airframe.jmx:name=named", "memory.free")
```

Comment:
Where does this `named` value come from?

Reply:
Oops, it was introduced when I was testing explicit naming for MBean class itself. Fixed.